src/app/core/components/info-card/info-card.component.scss

```diff
@@ -0,0 +1,107 @@
+@import "../../../shared/variables/global.scss";
+
+// Variables
+$card-width: 28rem;
```

Comment:
Variables you can extract to separate file

Reply:
8e4a0ffb0edb72372ade2f9c8432cf8130b128da